### DISCUSSION TITLE
d2ClanTracker

### DISCUSSION THREAD

**Discussion by cleverNamesAreHard** [nxtlo/aiobungie#419] [Show and tell]
Hey, folks.  Your library has helped me spin up what I'm hoping will be a source for folks to pull aggregate clan reports for activity completion.  Truly amazing work.  Your docs are great, and they helped me get something together in a tenth the time it would have taken me by fighting with the API directly.

Public Repo: [d2ClanTracker](https://github.com/cleverNamesAreHard/D2ClanTracker)
Public Site: https://www.d2clantracker.com

I apologize for the site being so slow.  I need to update the pagination, and I reckon there's bound to be something else I can do to speed up load time

Thanks, folks.

**Comment by nxtlo** [comment]
nice stuff! I'm glad aiobungie helped you with your application and i hope you can achieve whatever you're working on.

feel free to ask any questions
